README.md

```diff
@@ -41,6 +41,7 @@
 - 🇵🇭 Filipino
 - 🇫🇷 French
 - 🇩🇪 German
+- GR Greek
```

Comment:
Minor comment, but can you change this to use the emoji flag?

Reply:
I didn't know this was a thing 😇 
I think it's ok now 😃